actions.json

```diff
@@ -0,0 +1,20 @@
+[{
+        "id": "update_turnserver_ip",
+        "name": "Update turnserver ip",
+        "command": "/opt/yunohost/matrix-$YNH_APP_ID/Coturn_config_rotate.sh",
```

Comment:
As the app is multi instance, you should probably use YNH_APP_INSTANCE_NAME instead of YNH_APP_ID

Reply:
Fixed